setup.py

```diff
@@ -13,7 +13,8 @@
     "atomicwrites>=1.0",
     'funcsigs>=1.0;python_version<"3.0"',
     'pathlib2>=2.2.0;python_version<"3.6"',
-    'colorama<=0.4.1;sys_platform=="win32"',
+    'colorama<=0.4.1;sys_platform=="win32";python_version=="3.4"',
+    'colorama;sys_platform=="win32";python_version>"3.4"',
```

Comment:
this is missing python 2.7

Reply:
Would `python_version<="2.7",>"3.4"` be okay, or would `<"3"` be a better for handling python 2.7?

Given Python 2 is EOL in under a month, a 2.8 version shouldn't appear...

Reply:
`python_version<="2.7",>"3.4"` seems right to me. 👍

Reply:
@nightlark this branch keeps 2.7 support, mainline already dropped the old pythons

Reply:
Why 2.7, given that colorama 0.4.3 still runs there?!

Reply:
```python
    'colorama<=0.4.1;sys_platform=="win32";python_version=="3.4"',
    'colorama;sys_platform=="win32";python_version!="3.4"',
```

?

Reply:
`python_version!="3.4"` seems reasonable given that lower python 3.x versions were never supported in the 4.x releases.

Reply:
Okay, made the change (and fixed the syntax to use multiple environment markers for the conditions).

Reply:
> Do you want the new commit squashed into the old one for a cleaner history?

Yes, but took the liberty of doing it myself. 😁 

Thanks for the PR and patience! 👍